### ISSUE TITLE
Crashed when putting ae wireless terminal into bauble slot

### ISSUE THREAD

**Issue by V0odoofyxen** [AllTheMods/ATM3-Remix#153]
(cross posted to AE2) Playing All the mods 3 remix v 1.1.0 . Crashed upon putting wireless crafting terminal which had been combined with the fluid terminal and crafting terminal into bauble slot. reloaded/reentered the world, seemed to load fine but am unable to open my inventory at all, crashes as soon as I try to open it or use the key I assigned to open the terminal with.

any help would be appreciated

first crash report - https://pastebin.com/R9fP1nij
second crash report- https://pastebin.com/4YWFxNNm
third crash report - https://pastebin.com/PwRjbrae
last crash report - https://pastebin.com/Sp5pXWri

- Minecraft Version: 1.12.2
- AE2 Version:  appliedenergistics2 rv6 stable 6 jar is what it says in the mod list for ATM3r
- Forge Version: -14.23.5.2811

**Comment by Yamza** [comment]
https://github.com/ExtraCells/ExtraCells2/issues/641
Appears to be a known issue with no fix currently. You could try to edit it out of your inventory with an nbt editor, or restore a backup.

**Comment by V0odoofyxen** [comment]
thank for the quick reply.
is there an easy to use nbt editor, and which file in particular, the .ivio file in the playerdata file in the saves folder for the instance?

**Comment by Yamza** [comment]
Grab the latest release of NBTExplorer here: https://github.com/jaquadro/NBTExplorer/releases
extract it somewhere and open it. navigate to the proper save folder for your world, the into `level` and `playerdata` and click open.
then look for the inventory tag, find where the crafting terminal is and delete it. 
gif of removal: https://i.imgur.com/SuOPQEl.gifv

**Comment by V0odoofyxen** [comment]
finally managed to get to today after work, I had apparently at some point installed the latest nbtexplorer, though I don't remember doing it. it took a bit to find the terminal as I had a comply full inventory, it actually managed to save in the baubles inv, took me a couple of tries, it crashed again the first 2 times I deleted it, not sure if I saved the changes the first time, the second time I know saved and figured out it was saved somewhere else as well and restoring the bauble inv. 

that worked, thanks for the help.

 my last back up was a couple of hours back and I'd just finished a big chunk of work/clearing out the knight maze in tw forest and i wasnt looking forward to redoing it.
